mage_ai/streaming/sources/activemq.py

```diff
@@ -0,0 +1,87 @@
+import time
+from dataclasses import dataclass
+from typing import Callable
+
+import stomp
+from stomp.exception import ConnectFailedException
+
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.sources.base import BaseSource
+
+
+@dataclass
+class ActiveMQConfig(BaseConfig):
+    connection_host: str
+    connection_port: int
+    queue_name: str
+    configure_consume: bool = False
+    username: str = 'admin'
+    password: str = 'admin'
+
+
+def messageProcessingFunction(message, handler):
+    print('Recieved message: "%s"' % message)
+    handler(message)
```

Comment:
Usually, the handler expects the input to be a list of messages: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/streaming/sinks/base.py#L42

Reply:
Its not very clear from the stomp library whether **batch_read** is even possible. But given that ActiveMQ is not a designed to be a distributed system, it won't have extreme heavy loads, and hence, it should be ok to have handler called for each message.